src/ui/src/builder/settings/BuilderFieldsBlueprintId.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <div class="BuilderFieldsBlueprintId" :data-automation-key="props.fieldKey">
+        <WdsTextInput :model-value="blueprintId" disabled />
+    </div>
+</template>
+
+<script setup lang="ts">
+import { inject, computed, toRefs } from 'vue';
+import injectionKeys from '@/injectionKeys';
+import WdsTextInput from '@/wds/WdsTextInput.vue';
+
+const wf = inject(injectionKeys.core);
+
+const props = defineProps({
+    componentId: { type: String, required: true },
+    fieldKey: { type: String, required: true },
+});
+const { componentId } = toRefs(props);
+
+const component = computed(() => wf.getComponentById(componentId.value));
```

Comment:
Just in case you're not aware, you can also simplify to

```suggestion
const component = computed(() => wf.getComponentById(props.componentId));
```